docs/BackupAndRestore.md

```diff
@@ -0,0 +1,594 @@
+- `Archive` =  The persistent storage on disk of a NuoDB database served by a Storage Manager process (SM).
+A single NuoDB archive is represented by an archive directory and journal directory located on persistent storage.
+In Kubernetes, the journal directory is internal to the archive and normally located inside the archive directory.
+Each archive on disk has metadata representation in the admin layer uniquely identified by its `archiveId`.
+- `Hot copy` = Online backup for a database archive. This means taking a backup while the database is running.
+- `Backup set` = Hot copies are organized into backup sets. Backup sets provide a convenient mechanism for identifying hot copies and maintaining metadata information. A backup set is a directory created when initiating a full hot copy.  
+- `Offline backup` = A copy of a database archive taken while the Storage Manager (SM) process serving the archive is stopped. The copy/snapshot should include both the NuoDB archive and the NuoDB journal in a transactionally-consistent manner.
+- `HC SM` = Storage Manager engine pod controlled by NuoDB hot copy statefulset.
+- `nonHC SM` = Storage Manager engine pod controlled by NuoDB non-hotcopy statefulset.
+- `Database in-place restore` = A database restore that allows the data stored in the current database to go back in time without the need to provision additional resources (like more database processes, storage, etc).
+In most cases, this type of restore can be used in catastrophic failure situations, where restore is performed from the last good backup.
+- `Coordinator SM` = Storage Manager that is selected to prepare the domain state for database in-place restore.
+The coordinator SM will delete not restored archives and wait for all other archives to be restored before starting. 
+All other database processes wait for the coordinator SM to start before continuing their start up.
+- `Backup group` = A subset of Storage Managers in a database to which a backup request is sent by a scheduled backup job.
+Typically all HC SMs deployed in a single cluster are part of one backup group.
+- `Named backup group`= Backup group that has a custom name configured by the user.
+- `Seed restore` = A restore that allows a single archive to be restored using a backup. The archive is then updated with the latest transactions and brought current through the database _SYNCing_ process from a RUNNING SM.
+Normally used to speed up the SYNC process for the so-called "accelerated" sync use case.
+
+## Compatibility Matrix
+
+The table below shows the compatibility matrix for different backup and restore features in combination with the NuoDB product version and NuoDB Helm Charts.
+
+|                                       | NuoDB 4.0.x+ Helm Charts 2.x.x | NuoDB 4.0.x+ Helm Charts 3.0.x | NuoDB 4.0.x+ Helm Charts 3.1.x | NuoDB 4.2.x+ Helm Charts 3.2.x+ |
```

Comment:
Can we make it clear somewhere that this documents functionality of the last column, and there is no previous equvilent document?

Reply:
I would say that some parts of the document are specific to the last column but not all. For example, the _Scheduled online database backups_, _Automatic archive initial import_, and _Automatic archive restore_ haven't been changed a lot since the original implementation.
This compatibility matrix was my attempt to show what is supported where and to point major changes in the B&R logic.
I agree that the examples and the sample output is specific to the latest column so I will call that out.